svcs/ctfd/flagpool.go

```diff
@@ -87,21 +87,17 @@ func (fp *FlagPool) GetTagByIdentifier(id int) (store.Tag, error) {
 	return conf.Tag, nil
 }
 
-func (fp *FlagPool) TranslateFlagForTeam(t store.Team, cid int, value string) string {
+func (fp *FlagPool) TranslateFlagForTeam(t *store.Team, cid int, value string) string {
 	fp.m.RLock()
 	defer fp.m.RUnlock()
 
 	chal, ok := fp.ids[cid]
 	if !ok {
-		return ""
-	}
-
-	if chal.IsStatic() {
```

Comment:
that's why we do not have to check whether it is static or dynamic. I think.

Reply:
It depends on how the IsCorrectFlag handles static flags. We should test that before merging.

Reply:
yess, I do not have chance to see flags by using Kali linux, however I logged them to console, when challenges added to team. 
Eventually, all dynamic flags and static flags worked correctly !

Reply:
I just tested it an dynamic flags seem to be working :-D

However when doing it from a challenge that had been in the queue( no labs available), it did not work. 
I think this is because of the pointer in the handler not being valid(see my comment below).

Reply:
:(